apps/src/weblab2/Weblab2View.tsx

```diff
@@ -19,31 +18,27 @@ const weblabLangMapping: {[key: string]: LanguageSupport} = {
   css: css(),
 };
 
-const DefaultEditorComponent = () =>
-  CDOEditor(weblabLangMapping, ['html', 'css']);
-
 const horizontalLayout = {
   gridLayoutRows: '32px 300px auto',
   gridLayoutColumns: '300px auto auto',
-  gridLayout: `    "instructions file-tabs preview-container"
-      "instructions editor preview-container"
-      "file-browser editor preview-container"`,
+  gridLayout: `    "instructions workspace preview-container"
+      "instructions workspace preview-container"
+      "file-browser workspace preview-container"`,
 };
 
 const verticalLayout = {
   gridLayoutRows: '32px 300px auto auto',
   gridLayoutColumns: '300px auto',
-  gridLayout: `    "instructions file-tabs file-tabs"
-      "instructions editor editor"
-      "file-browser editor editor"
+  gridLayout: `    "instructions workspace workspace"
+      "instructions workspace workspace"
+      "file-browser workspace workspace"
```

Comment:
same as the PythonLab comments above, we can simplify the `gridLayoutRows` and redundant `gridLayout` line.